### ISSUE TITLE
lxc import fails due to NIC device having different subnet than managed bridge

### ISSUE THREAD

**Issue by tomponline** [lxc/lxd#10113]
See https://discuss.linuxcontainers.org/t/lxc-import-fails-due-to-lxdbr0-having-different-subnet/13639

We probably need to relax/ignore failures of the import time device validation (similar to when we restore a snapshot) to allow import and manual correction of the problem before starting.

Or we need to add a `-d` flag to `lxc import` similar to that of `lxc copy` to allow users to make import-time device modification.

The latter option seems safer as it avoids the risk of corrupting another instance's static DHCP allocation in the scenario of importing an instance with the same IP as an existing instance. See related https://discuss.linuxcontainers.org/t/unable-to-copy-container-due-to-already-defined-ip-address/13654

If we go that option then we should probably also do the same for `lxc restore` for consistency and safety (in case restoring a snapshot causes an IP conflict similar to doing `lxc import` or `lxc copy`) rather than ignoring the problem at restore time.

**Comment by stgraber** [comment]
Closing in favor of the other more generic issue which covers this too.

@tomponline can you edit that one to cover anything it's missing from this one?
